<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicketCard</class>
 <widget class="QWidget" name="TicketCard">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="ticketTitleLabel">
     <property name="text">
      <string>Заголовок тикета</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="ticketDescription">
     <property name="placeholderText">
      <string>Описание тикета...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="infoBox">
     <property name="title">
      <string>Информация</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0"><widget class="QLabel"><property name="text"><string>Проект</string></property></widget></item>
      <item row="0" column="1"><widget class="QLabel" name="labelProjectValue"/></item>
      <item row="1" column="0"><widget class="QLabel"><property name="text"><string>Трекер</string></property></widget></item>
      <item row="1" column="1"><widget class="QLabel" name="labelTrackerValue"/></item>
      <item row="2" column="0"><widget class="QLabel"><property name="text"><string>Название</string></property></widget></item>
      <item row="2" column="1"><widget class="QLabel" name="labelTitleValue"/></item>
      <item row="3" column="0"><widget class="QLabel"><property name="text"><string>Статус</string></property></widget></item>
      <item row="3" column="1"><widget class="QLabel" name="labelStatusValue"/></item>
      <item row="4" column="0"><widget class="QLabel"><property name="text"><string>Приоритет</string></property></widget></item>
      <item row="4" column="1"><widget class="QLabel" name="labelPriorityValue"/></item>
      <item row="5" column="0"><widget class="QLabel"><property name="text"><string>Назначена</string></property></widget></item>
      <item row="5" column="1"><widget class="QLabel" name="labelAssignedValue"/></item>
      <item row="6" column="0"><widget class="QLabel"><property name="text"><string>Наблюдатель</string></property></widget></item>
      <item row="6" column="1"><widget class="QLabel" name="labelObserverValue"/></item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="filesBox">
     <property name="title">
      <string>Файлы</string>
     </property>
     <layout class="QVBoxLayout" name="filesLayout">
      <item>
       <widget class="QListWidget" name="fileList"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="historyText">
     <property name="placeholderText">
      <string>История изменений...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="commentEdit">
     <property name="placeholderText">
      <string>Комментарий редактирования</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
